### PR TITLE
Editorial: additions to enctype description

### LINKED ISSUE #1445: no description enctype attribute of form element
no description of the attribute **enctype** of form element what is and how works is:
"application/x-www-form-urlencoded"
"multipart/form-data"
"text/plain"

### PR DESCRIPTION
**Pull request by scottaohara**
Fixes #1445 

Adding some additional context to the purpose of the enctype attribute.

Converted the list of different keywords/states to a `dl` to provide short descriptions of each.

### CODE REVIEW

File: sections/semantics-forms.include [hunk 1]
```diff
@@ -10489,29 +10489,53 @@ You cannot submit this form when the field is incorrect.</samp></pre>
...
+    <dd>
+      The default value if the attribute is not specified.
+    </dd>
+
+    <dt>
+      The "<dfn attr-value for="form/enctype"><code>multipart/form-data</code></dfn>"
+      keyword and corresponding state.
+    </dt>
+    <dd>
+      Allows for entire files to be submitted along with with form data.
```
Comment (siusin) [inline sections/semantics-forms.include]: There is a duplicate "with" here.

File: sections/semantics-forms.include [hunk 2]
```diff
@@ -10489,29 +10489,53 @@ You cannot submit this form when the field is incorrect.</samp></pre>
...
+    </dd>
+
+    <dt>
+      The "<dfn attr-value for="form/enctype"><code>text/plain</code></dfn>"
+      keyword and corresponding state.
+    </dt>
+    <dd>
+      Submits as <a>plain text form data</a>, which is meant to be human readable.
```
Comment (siusin) [inline sections/semantics-forms.include:10510]: This anchor doesn't work... if it's defined in `<h5 id="plain-text-form-data">Plain text form data</h5>`, perhaps we should use `<a href="#plain-text-form-data">` instead.

**Review by siusin — COMMENTED** [review]
Looks good to me except for two minor editorial issues.

File: sections/semantics-forms.include (hunk 2 above)
Comment (chaals) [reply sections/semantics-forms.include:10510]: or make the term there a definition: `<h5><dfn>plain...`

**Pull request merged**